src/types.tsx

```diff
@@ -101,8 +100,8 @@ export type GestureResponseDistanceType = {
 export type SearchBarPlacement = 'automatic' | 'inline' | 'stacked';
 
 export interface ScreenProps extends ViewProps {
-  active?: 0 | 1 | Animated.AnimatedInterpolation<number>;
-  activityState?: 0 | 1 | 2 | Animated.AnimatedInterpolation<number>;
```

Comment:
Not sure about this. Does the Animated value even work with `activityState`?

Reply:
I think react navigation uses this. Why would we remove this, what's the reasoning here?

Reply:
The problems were caused with wrong types compatibility layer being removed in: https://github.com/software-mansion/react-native-screens/pull/2423/commits/cb9ba27ff2aa550346d75a4174bc3ac3374325b9